src/ngraph/runtime/ie/CMakeLists.txt

```diff
@@ -30,11 +30,11 @@ set(SRC
     ie_executable.hpp
 )
 
+find_package(InferenceEngine REQUIRED)
 add_library(ie_backend ${LIBRARY_TYPE} ${SRC})
-add_dependencies(ie_backend inference_engine)
 target_compile_definitions(ie_backend PRIVATE IE_BACKEND_DLL_EXPORTS)
-target_include_directories(ie_backend PUBLIC ${IE_MAIN_SOURCE_DIR}/include)
-target_link_libraries(ie_backend PUBLIC ngraph inference_engine)
+target_include_directories(ie_backend PUBLIC ${InferenceEngine_INCLUDE_DIRS})
```

Comment:
I guess you can remove this line and don't add `${InferenceEngine_INCLUDE_DIRS}` CMake automatically adds this folders from `${InferenceEngine_LIBRARIES}`